### PR TITLE
Changed the HD Profile Picture value - user_info_by_username_v1

### PR DESCRIPTION
**Pull request by ashirali4**
I have added this code 

image

Instead of this 

image
Fetching the hd version from the second one first, anyways I am new to py and flask so unable to find it in `user_info_by_username()`

Anyways I have added here `user_info_by_username_v1()`

Do check and let me know what we can do @adw0rd

### CODE REVIEW

File: instagrapi/extractors.py [hunk 1]
```diff
@@ -188,9 +188,9 @@ def extract_user_gql(data):
 def extract_user_v1(data):
     """For Private API"""
     data["external_url"] = data.get("external_url") or None
-    pic_hd = data.get("hd_profile_pic_url_info") or data.get("hd_profile_pic_versions")
+    pic_hd = data.get("hd_profile_pic_versions") or data.get("hd_profile_pic_url_info")
     if pic_hd:
-        data["profile_pic_url_hd"] = pic_hd.get("url")
+        data["profile_pic_url_hd"] = pic_hd[1].get("url") or pic_hd.get("url")
```
Comment (adw0rd) [inline instagrapi/extractors.py:193]: Тут будет ошибка в случае `pic_hd = data.get("hd_profile_pic_url_info")`, так как это не список, а словарь

Comment (adw0rd) [inline instagrapi/extractors.py:193]: Why `pic_hd[1]`?  Maybe `pic_hd[2]`?

Maybe you need to sort first? For example like this:

`sorted(pic_hd, key=lambda o: o["height"] * o["width"])[-1]["url"]`

**Review by adw0rd — CHANGES_REQUESTED** [review]

File: instagrapi/extractors.py (hunk 1 above)
Comment (ashirali4) [reply instagrapi/extractors.py:193]: I have reviewed them, and they are already sorted on the basis of height and width.

Comment (ashirali4) [reply instagrapi/extractors.py:193]: and getting max 0,1 two image object not more than that.

Comment (ashirali4) [reply instagrapi/extractors.py:193]: I'll fix this as the profile_pic_url_hd is an array and otherone is dict.

**Review by ashirali4 — COMMENTED** [review]
There will be an error in the case of pic_hd = data.get("hd_profile_pic_url_info"), since this is not a list, but a dictionary - Fixed.

File: instagrapi/extractors.py (hunk 1 above)
Comment (ashirali4) [reply instagrapi/extractors.py]: image

**Comment by ashirali4** [comment]
any update @adw0rd

**Comment by adw0rd** [comment]
@ashirali4 I rewrote your code

https://github.com/adw0rd/instagrapi/commit/7cfaaef26421edb32d3fbbc3e4352afd02eef2fd

**Pull request merged**